bench/ArrayBench.java

```diff
@@ -0,0 +1,30 @@
+package benchmark;
+
+import com.sun.cldchi.jvm.JVM;
+
+public class ArrayBench {
+    void runBenchmark() {
+        long start, time;
+        int len = 1;
+        int iterations = 1000;
+
+        System.out.println("Time to allocate int arrays of various lengths " + iterations + " times, by length, in ms:");
+
+        for (int i = 0; i < 20; i++) {
+
+            start = JVM.monotonicTimeMillis();
+            for (int j = 0; j < iterations; j++) {
```

Comment:
To see differences when the length of the array is small, we could use a larger number of iterations.